.eslintrc.json

```diff
@@ -30,10 +27,42 @@
     "testing-library/prefer-explicit-assert": "warn",
     "testing-library/prefer-presence-queries": "error",
     "testing-library/prefer-screen-queries": "error",
-    "testing-library/prefer-wait-for": "error",
-    "no-use-before-define": "off",
-    "@typescript-eslint/no-use-before-define": [
-      "error"
-    ]
-  }
+    "testing-library/prefer-wait-for": "error"
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts", "**/*.tsx"],
+      "env": {"browser": true, "es6": true, "node": true},
+      "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"
+      ],
+      "globals": {"Atomics": "readonly", "SharedArrayBuffer": "readonly"},
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "ecmaFeatures": {"jsx": true},
+        "ecmaVersion": 2018,
```

Comment:
will this accept `esnext` instead of 2018?
Guessing 2018 was around the time the SO post was created

Reply:
Doesn't look like it: 
`Value is not accepted. Valid values: 3, 5, 6, 2015, 7, 2016, 8, 2017, 9, 2018, 10, 2019, 11, 2020, 12, 2021.`

Reply:
2021 it is 🚀